src/jetpack-editor-setup.js

```diff
@@ -19,7 +19,7 @@ const supportedJetpackBlocks = {
 	},
 };
 
-const setJetpackData = async ( {
```

Comment:
Without removing this, we would start parsing html to blocks before the jetpack blocks got registered. I'm not sure if this changes our existing thoughts on when we should check for jetpack permission. Perhaps it should be done on the app side. But it will at least have to block the parsing of HTML to blocks otherwise existing jetpack blocks will show up as unknown blocks.

Reply:
Agreed block parsing (and thus loading the editor) should be done afterwards. We can delay the loading of the editor for a bit but it wouldn't be a great UX if it happened each time. It would be nice if we could get jetpack data from the parent app, or at least cache here it somehow.